src/ergw_aaa_diameter.erl

```diff
@@ -150,6 +152,17 @@ validate_transport(Opt, Value) ->
 validate_transport_error(Opt, Value) ->
     throw({error, {options, {Opt, Value}}}).
 
+validate_handler(Opts) ->
+    ergw_aaa_config:validate_options(fun validate_option/2, Opts, ?DefaultOptions, map).
+
+validate_option(termination_cause_mapping = Opt, Value) when is_list(Value) ->
+    case [{N, V} || {N, V} <- Value, is_atom(N), is_integer(V)] of
```

Comment:
and adding to the above comment, this should use `ergw_aaa_config:validate_options`

Reply:
But I already use `ergw_aaa_config:validate_options` in `validate_handler/1` and `validate_option` this is private function. Can you provide example what you expected here?

Reply:
Done